src/plugins/vis_types/timeseries/public/application/components/vis_types/top_n/vis.js

```diff
@@ -32,7 +32,7 @@ function sortSeries(visData, model) {
   const series = get(visData, `${model.id}.series`, []);
   return model.series.reduce((acc, item) => {
     const itemSeries = series.filter((s) => {
-      const id = first(s.id.split(/:/));
+      const id = first(s.id.split(/╰┄►/));
```

Comment:
why do we use constant above and put the symbol inline in the rest of the files? 🤔

Reply:
Good point, changed these to use the constant